tests/conftest.py

```diff
@@ -19,16 +19,36 @@ def set_api_key(monkeypatch):
     monkeypatch.setattr(workos, "api_key", "sk_abdsomecharactersm284")
 
 
+@pytest.fixture
+def set_environment_id(monkeypatch):
+    monkeypatch.setattr(workos, "environment_id", "environment_b27needthisforssotemxo")
+
+
+@pytest.fixture
+def set_client_id(monkeypatch):
+    monkeypatch.setattr(workos, "client_id", "client_b27needthisforssotemxo")
+
+
 @pytest.fixture
 def set_project_id(monkeypatch):
     monkeypatch.setattr(workos, "project_id", "project_b27needthisforssotemxo")
 
 
+@pytest.fixture
+def set_api_key_and_client_id(set_api_key, set_client_id):
+    pass
+
+
 @pytest.fixture
 def set_api_key_and_project_id(set_api_key, set_project_id):
     pass
 
 
+@pytest.fixture
+def set_api_key_and_environment_id(set_api_key, set_environment_id):
```

Comment:
Do we need this?

Reply:
With the clarification of not needing env_id for the AuditTrail test, no we wouldn't need this fixture!

Reply:
Actually, @rjvani, if this is true for passwordless as well (https://github.com/workos-inc/workos-python/blob/feature/ch4527-32/sdk-updates-for-dashboard2/tests/test_passwordless.py), we could remove project_id fixtures as well!